docs/compendium/substitute.md

```diff
@@ -1,19 +1,30 @@
 # Substitute compendia
 
-introduction...
+Introduction to substitution of two compendia. <br/>
```

Comment:
Don't use HTML tags that you can easily do by adding an empty line in plain markdown.

Also, it obvious the introduction is in the beginning. Suggest to start with: "Substitution is the re-combination of a base compendium and and overlay compendium."